plugins/scss/index.js

```diff
@@ -154,14 +159,54 @@ const writer = function(thisPackage, outputDir) {
       file = path.resolve(outputDir, 'combined.scss');
       args.push(file);
       console.log('Writing ' + file);
-      promises.push(concat(scssEntries, file));
+      promises.push(concat.concatAsync(scssEntries, file));
     }
 
     file = path.resolve(outputDir, 'node-sass-args');
     console.log('Writing ' + file);
     promises.push(fs.writeFileAsync(file, args.join(' ')));
 
-    return Promise.all(promises);
+    return new Promise(function(resolve, reject) {
```

Comment:
Why the extra `new Promise()` wrapper here? Shouldn't you be able to simply return the `Promise.all(...)`?

Reply:
I think i was trying to workaround the writeFileAsync still working when the test was trying to clean it up